Sources/CasePaths/CasePathable.swift

```diff
@@ -272,7 +272,7 @@ extension CaseKeyPath {
 /// A partially type-erased key path, from a concrete root enum to any resulting value type.
 public typealias PartialCaseKeyPath<Root> = PartialKeyPath<Case<Root>>
 
-extension PartialCaseKeyPath {
+extension _AppendKeyPath {
```

Comment:
This is the same trick the standard library uses to avoid surfacing partial key path methods on non-partial key paths.